pandas/io/pytables.py

```diff
@@ -2490,8 +2475,7 @@ def get_atom_timedelta64(cls, shape):
 class GenericDataIndexableCol(DataIndexableCol):
```

Comment:
Can we just blow this class away altogether now?

Reply:
worth a shot